client/my-sites/domains/domain-management/transfer/transfer-out/locked.jsx

```diff
@@ -80,7 +80,7 @@ class Locked extends React.Component {
 						primary
 						disabled={ this.state.submitting }
 					>
-						{ translate( 'Update Settings And Continue' ) }
+						{ translate( 'Update settings and continue' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.